packages/babel-plugin-transform-es2015-modules-systemjs/test/fixtures/systemjs/exports-named/expected.js

```diff
@@ -18,12 +20,8 @@ System.register([], function (_export, _context) {
 
       _export("bar", bar);
 
-      function foo() {}
-
       _export("foo", foo);
```

Comment:
This export should really move along with the function hoisted - as named function declarations are allowed bindings in ES module circular references.

Let me know if that makes sense?